packages/ui/src/SelectField.tsx

```diff
@@ -31,6 +31,41 @@ const Input = (props: any) => {
   return <components.Input {...props} autoComplete="chrome-off" />
 }
 
+// This component is implemented because we want to style things with the css (as opposed to the react-select
+// recommended way of styling things via js).
+// We simply inject `.multiValueIsFocused`.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const MultiValue = (props: MultiValueProps<any>) => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  props.components.Remove = MultiValueRemove
+  return <components.MultiValue {...props} className={cn({ [styles.multiValueIsFocused]: props.isFocused })} />
+}
+
+// Self styled version of the MultiValue/Remove button
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const MultiValueRemove = (props: any) => {
+  // We have to pass down the onClick and onMouseDown handler to our IconButton
+  // otherwise a click is a noop.
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,  @typescript-eslint/no-unsafe-assignment
+  const onClick: React.MouseEventHandler = props.innerProps.onClick
```

Comment:
Can't we stop propagation for the onClick and not introduce the mouseDown handler?

Reply:
nope, that doesn't work. it was the MouseDown chain of events that should be stopped.